client/scripts/views/modals/select_address_modal.ts

```diff
@@ -122,6 +121,12 @@ const SelectAddressModal: m.Component<{}, { selectedIndex: number, loading: bool
             ]),
           ])),
         ]),
+        !!meta.terms
```

Comment:
we should standardize this check across the files touched in this PR. It should just be consistent

Reply:
SG! On it